eva-server/src/main/java/uk/ac/ebi/eva/server/ws/contigalias/ContigAliasService.java

```diff
@@ -157,12 +178,73 @@ private String translateContigNameToInsdc(String contigName, String assembly, Co
         return ContigAliasTranslator.getTranslatedContig(contigAliasResponse, ContigNamingConvention.INSDC);
     }
 
+    public List<ContigAliasChromosome> searchChromosomeByName(String contigName, String assembly, ContigNamingConvention contigNamingConvention) {
+        String url = contigAliasUrl + CONTIG_ALIAS_CHROMOSOMES_SEARCH_ENDPOINT + contigName;
+        if ((assembly != null && !assembly.isEmpty()) && contigNamingConvention != null) {
+            url += "?assemblyAccession=" + assembly + "&namingConvention=" + getNameParam(contigNamingConvention);
+        } else if (assembly != null && !assembly.isEmpty()) {
+            url += "?assemblyAccession=" + assembly;
+        } else if (contigNamingConvention != null) {
+            url += "?namingConvention=" + getNameParam(contigNamingConvention);
+        }
+
+        ContigAliasResponse contigAliasResponse = restTemplate.getForObject(url, ContigAliasResponse.class);
+        if (contigAliasResponse == null || contigAliasResponse.getEmbedded() == null ||
+                contigAliasResponse.getEmbedded().getContigAliasChromosomes() == null ||
+                contigAliasResponse.getEmbedded().getContigAliasChromosomes().isEmpty()) {
+            return null;
+        }
+
+        return contigAliasResponse.getEmbedded().getContigAliasChromosomes();
+    }
+
+    public ContigAliasChromosome getUniqueInsdcChromosomeByName(String contigName, String assembly, ContigNamingConvention contigNamingConvention) {
+        List<ContigAliasChromosome> chromosomeList = searchChromosomeByName(contigName, assembly, contigNamingConvention);
```

Comment:
Because we're likely going to make this query many time and this will never change, is it possible to cache it?
Maybe a different ticket though.